index.html

```diff
@@ -5,8 +5,46 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet"/>
+    <script src="./node_modules/axios/dist/axios.min.js"></script>
 </head>
 <body>
-    
+    <header class="header">
+        <h1 id="topHeader">!!!</h1>
+    </header>
+    <main class="container">
+        <div id="temperature">
+            <h2>Temperature</h2>
+            <button onclick="increment()">⬆️</button>
+            <h2 id="root"></h2>
```

Comment:
Style: Remember, tags like `h2` should be used to title parts of your webpage.  This part already has a `h2` and this `h2` is empty.